src/main/java/lotto/game/LottoCount.java

```diff
@@ -1,30 +1,65 @@
 package lotto.game;
 
 import lotto.money.Money;
+import lotto.ticket.Ticket;
 
 import java.util.Objects;
 
+import static lotto.ticket.Number.ERROR_MESSAGE_INVALID_INPUT;
+import static lotto.ticket.Number.validateNumber;
+
 public class LottoCount {
+    public static final String ERROR_MESSAGE_INVALID_AMOUNT = "구매 금액보다 많이 구입할 수 없습니다.";
     public static final int ZERO = 0;
     public static final int ONE_COUNT = 1;
-    private static final int LOTTO_PRICE = 1000;
 
     private final int lottoCount;
 
     public LottoCount(Money money) {
-        this.lottoCount = money.divideMoney(LOTTO_PRICE);
+        this(money.divideMoney(Ticket.PRICE));
+    }
+
+    public LottoCount(String value) {
+        this.lottoCount = validate(value);
     }
 
-    private LottoCount(int lottoCount) {
-        this.lottoCount = lottoCount;
+    private int validate(String value) {
+        int number = validateNumber(value);
+        validateNotNegative(number);
+        return number;
+    }
+
+    private void validateNotNegative(int number) {
+        if (number < ZERO) {
+            throw new IllegalArgumentException(ERROR_MESSAGE_INVALID_INPUT);
+        }
     }
 
     public boolean isGreaterThanZero() {
         return this.lottoCount > ZERO;
     }
 
     public LottoCount decreaseOne() {
-        return new LottoCount(lottoCount - ONE_COUNT);
+        return new LottoCount(String.valueOf(lottoCount - ONE_COUNT));
```

Comment:
```suggestion
        return new LottoCount(lottoCount - ONE_COUNT);
```

int 생성자도 제공해주면 어떨까요!?